interface/patient_file/history/track_appointments.php

```diff
@@ -0,0 +1,249 @@
+<?php
```

Comment:
Same here.

Reply:
This is a new file and is interface for track appointments tab. Will add header info.